src/widgets/Footer/components/OrderProcessing.tsx

```diff
@@ -0,0 +1,14 @@
+const OrderProcessing = () => {
+  return (
+    <div className="flex flex-col justify-start mb-13">
+      <div className="font-roboto text-xs font-medium text-text-tertiary">
```

Comment:
you should add a changes for colors. The duplication of "text-text" is not good

Reply:
We've to change it in tailwind.config